src/components/HeartFavorite/HeartFavorite.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Check how icons are set here: https://github.com/tepachelabs/songbox.webapp/pull/33/files#diff-0c98ae846874305539a312f8b3606023eb76035f89dda93a48b9f0e6e6a169b0R1

Please try to follow the same pattern.